src/test/assets/meson/some.meson.Project.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id": "some.meson.Project",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "meson",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "meson",
+            "builddir": true,
```

Comment:
I don't think we actually handle the builddir argument in flatpak-vscode it is mostly used by builder to whether build stuff in current directory or do so in the cache directory.